runner/components/io.siddhi.distribution.core/src/main/java/io/siddhi/distribution/core/internal/ServiceComponent.java

```diff
@@ -166,6 +168,22 @@ protected void start(BundleContext bundleContext) throws Exception {
                 log.error("Error: Can't get target file to run. System property {} is not set.",
                         SiddhiAppProcessorConstants.SYSTEM_PROP_RUN_SIDDHI_APPS);
             } else {
+                // Change relative paths to absolute
+                Path siddhiAppsGivenPath = Paths.get(siddhiAppsReference);
+                if (!siddhiAppsGivenPath.isAbsolute()) {
+                    if (System.getProperty(
+                            SiddhiAppProcessorConstants.SYSTEM_PROP_CURRENT_DIRECTORY
+                    ) != null) {
+                        Path currentWorkingDirectory = Paths.get(
+                                System.getProperty(
+                                        SiddhiAppProcessorConstants.SYSTEM_PROP_CURRENT_DIRECTORY
+                                )
+                        ).toAbsolutePath();
+                        siddhiAppsReference = currentWorkingDirectory.resolve(
```

Comment:
Seems there are additional/unwanted line breaks. Please check if so.

Reply:
Fixed in here https://github.com/siddhi-io/distribution/pull/798/commits/f14c8165ffaecb7471ab7ce7e2918aef2798ab4b